packages/shared/babel-ast-utils/src/index.js

```diff
@@ -24,6 +25,42 @@ import {generator, expressionsPrecedence} from './generator';
 
 export {babelErrorEnhancer};
 
+export function remapAstLocations(ast: BabelNodeFile, map: SourceMap) {
+  // remap ast to original mappings
+  // This improves sourcemap accuracy and fixes sourcemaps when scope-hoisting
```

Comment:
I guess this is only used if you are using babel and there's an existing sourcemap now?

Reply:
Yes